app/forms/individual/EmailAddressForIndividualFormProvider.scala

```diff
@@ -27,9 +27,11 @@ class EmailAddressForIndividualFormProvider @Inject() extends Mappings with Rege
 
   def apply(): Form[String] =
     Form(
-      "email" -> validatedTextMaxLength(
+      "email" -> validatedText(
         "emailAddressForIndividual.error.required",
+        "emailAddressForOrganisation.error.invalid",
```

Comment:
@iservport The error message key "emailAddressForOrganisation.error.invalid" should it not be Individual one